ui/core/src/users/List.js

```diff
@@ -0,0 +1,80 @@
+// file: users/List.js
+
+import React, { PropTypes } from 'react';
+import Relay from 'react-relay';
+import { withRouter, Link } from 'react-router';
+
+// Base components
+import Grid, { Cell } from 'react-to-mdl/grid';
+import Textfield from 'react-to-mdl/textfield';
+import Select from 'react-select';
+import Button from 'react-to-mdl/button';
+import DataTable, {
+  DataTableHead,
+  DataTableTH,
+  DataTableTD
+} from 'react-to-mdl/dataTable';
+import Layout, { LayoutIcon } from 'react-to-mdl/layout';
+
+// CSS
+import CSSModules from 'react-css-modules';
+import { mdlUpgrade } from 'react-to-mdl';
+import styles from './styles.scss';
+
+class List extends React.Component {
+
+  static defaultProps = {
+    title: 'Lista de Usuarios'
+  }
+
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    const { title, users } = this.props;
+
+    return (
+      <div>
+        <Grid>
+          <Cell col={12}>
+            <h1 styleName="form-title">{title}</h1>
+          </Cell>
+        </Grid>
+        <Grid>
+          <Cell offset={3} col={6}>
+            <DataTable shadow={'2dp'}>
+              <DataTableHead>
+                <DataTableTH nonNumeric={true}> Usuario </DataTableTH>
```

Comment:
Please use english for this kind of _labels_ or _headers_ here an throughout this project